pkgs/by-name/bk/bkyml/package.nix

```diff
@@ -21,9 +21,9 @@ python3.pkgs.buildPythonApplication rec {
     substituteInPlace setup.py \
       --replace-fail "['pyscaffold>=3.0a0,<3.1a0'] + " "" \
       --replace-fail "use_pyscaffold=True"  ""
-    substituteInPlace src/bkyml/skeleton.py --replace-fail \
-        "from bkyml import __version__" \
-        "__version__ = \"${version}\""
+    substituteInPlace src/bkyml/__init__.py \
+      --replace-fail "from pkg_resources" "# from pkg_resources" \
+      --replace-fail "get_distribution(dist_name).version" '"${version}"'
```

Comment:
`__init__` is automatically imported anyways even if `__version__` was not actually used, thus requiring setuptools during runtime.
Now instead of patching the usage of `__version__` we patch the value of `__version__` itself.